cmd/packagesfuncs.go

```diff
@@ -46,6 +46,25 @@ func cmdUpgrade(fl Flags) (int, error) {
 	return upgradeBuild(pluginPkgs, fl)
 }
 
+func splitModule(arg string) (module, version string, err error) {
+	const versionSplit = "@"
+
+	// accommodate module paths that have @ in them, but we can only tolerate that if there's also
+	// a version, otherwise we don't know if it's a version separator or part of the file path (see #109)
```

Comment:
What is `#109`? #109 ??

Reply:
This is actually a issue ID from xcaddy as i copied a large chunk of the split module function from there. I guess i missed this, ill remove it

Reply:
Ah, haha, I thought it looked familiar. Thanks